src/code/z_actor_dlftbls.c

```diff
@@ -38,45 +38,44 @@ ActorOverlay gActorOverlayTable[] = {
 
 ActorId gMaxActorId = 0;
 
-FaultClient D_801ED930;
-FaultAddrConvClient D_801ED940;
+FaultClient sActorOverlayTableFaultClient;
+FaultAddrConvClient sActorOverlayTableFaultAddrConvClient;
 
-void ActorOverlayTable_FaultPrint(void* arg0, void* arg1) {
+void ActorOverlayTable_FaultClient(void* arg0, void* arg1) {
     ActorOverlay* overlayEntry;
     u32 overlaySize;
-    ActorId i;
+    ActorId actorId;
 
     FaultDrawer_SetCharPad(-2, 0);
 
     FaultDrawer_Printf("actor_dlftbls %u\n", gMaxActorId);
     FaultDrawer_Printf("No. RamStart- RamEnd cn  Name\n");
 
-    for (i = 0, overlayEntry = &gActorOverlayTable[0]; i < gMaxActorId; i++, overlayEntry++) {
+    for (actorId = 0, overlayEntry = &gActorOverlayTable[0]; actorId < gMaxActorId; actorId++, overlayEntry++) {
         overlaySize = VRAM_PTR_SIZE(overlayEntry);
         if (overlayEntry->loadedRamAddr != NULL) {
-            FaultDrawer_Printf("%3d %08x-%08x %3d %s\n", i, overlayEntry->loadedRamAddr,
+            FaultDrawer_Printf("%3d %08x-%08x %3d %s\n", actorId, overlayEntry->loadedRamAddr,
                                (u32)overlayEntry->loadedRamAddr + overlaySize, overlayEntry->numLoaded, "");
         }
     }
 }
 
-void* ActorOverlayTable_FaultAddrConv(void* arg0, void* arg1) {
-    u8* ptr = arg0;
-    ActorOverlay* overlayEntry = &gActorOverlayTable[0];
-    ActorId i;
-    u8* ramStart;
-    u8* ramEnd;
-    size_t size;
-    u32 offset;
-
-    for (i = 0; i < gMaxActorId; i++, overlayEntry++) {
-        size = VRAM_PTR_SIZE(overlayEntry);
-        ramStart = overlayEntry->loadedRamAddr;
-        ramEnd = ramStart + size;
-        offset = (u8*)overlayEntry->vramStart - ramStart;
+void* ActorOverlayTable_FaultAddrConv(void* address, void* param) {
+    uintptr_t addr = address;
+    ActorOverlay* actorOvl = &gActorOverlayTable[0];
+    uintptr_t ramConv;
+    void* ramStart;
+    uintptr_t diff;
+    ActorId actorId;
+
+    for (actorId = 0; actorId < gMaxActorId; actorId++, actorOvl++) {
+        diff = VRAM_PTR_SIZE(actorOvl);
```

Comment:
What's the reason for changing this from `size_t` to `uintptr_t`?

Reply:
That is just what was there from the graph function, which is what I copied to the others. I do agree `size_t` fits better so will change. I also think `ramConv` also makes more sense as a `size_t` so will change that as well.